packs/csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl

```diff
@@ -0,0 +1,84 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+
+  datacenters = [[ .csi_openstack_cinder.datacenters | toPrettyJson ]]
+
+  type = system
+
+  group "nodes" {
+
+    restart {
+      attempts = [[ .csi_openstack_cinder.job_restart_config.attempts ]]
+      delay    = [[ .csi_openstack_cinder.job_restart_config.delay ]]
+      mode     = [[ .csi_openstack_cinder.job_restart_config.mode ]]
+      interval = [[ .csi_openstack_cinder.job_restart_config.interval ]]
+    }
+    
+    [[ template "constraints" .csi_openstack_cinder.constraints ]]
+
+    [[- template "vault_config" .csi_openstack_cinder -]]
```

Comment:
```suggestion
    [[- template "vault_config" .csi_openstack_cinder ]]
```

The trailing `-` here causes the newline we need before the `task` block to disappear, like so:

```
  constraint {
    attribute = "${attr.platform.aws.placement.availability-zone}"
    value     = "nova"
  }task "cinder-node" {
      driver = "docker"
      template {
        data        = <<EOT
```

This causes this error on the Nomad side:

```
    2021-11-05T18:09:48.261Z [DEBUG] http: request failed: method=POST path=/v1/jobs/parse error="input.hcl:23,4-8: Missing newline after block definition; A block definition must end with a newline." code=400
```

Reply:
Added